bylaws.tex

```diff
@@ -207,6 +207,7 @@
 	\item The applicant submits an application to the Evaluations Director for review.
 	\item The applicant participates in an informal interview with exactly three current Active, Alumni, or Honorary Members.
 	\item The application materials are then reviewed at an Evaluations meeting and the members present vote on whether to accept the person as an introductory member.
+	\item If the applicant has passed an Introductory Evaluation in the past an optional Simple Majority vote may be held to allow the applicant to automatically pass the Introductory Evaluation. Any member in attendance may motion for this vote to occur.
```

Comment:
Fair, I'll change that if it passes. That way it's not confusing for people when it's in-flight.